### DISCUSSION TITLE
Connection speed of offers. WEBRTC

### DISCUSSION THREAD

**Discussion by Gaspela** [bluenviron/mediamtx#3758] [Questions]
### Question

I wonder how they managed to manage the offers using webrtc so quickly? It currently takes me 9 seconds per request for connection offers.

**Comment by aler9** [comment]
If you still have problems, open another discussion and provide, as mentioned in instructions:

* the server version you are using
* precise instructions on how to replicate the problem
* server logs with setting logLevel set to debug
